databases/metadata/src/main/functions/04_ontology_hierarchy.sql

```diff
@@ -0,0 +1,32 @@
+SET search_path = public, pg_catalog;
+
+--
+-- A function that returns all of the ontology subclasses of the class with the given
+-- ontology_version and IRI.
+--
+CREATE OR REPLACE FUNCTION ontology_class_hierarchy(VARCHAR, VARCHAR)
```

Comment:
In the interest of keeping on top of performance, I'd love to see an `EXPLAIN ANALYZE` of this with some loaded data.